test/chi/components/header.pug

```diff
@@ -567,45 +497,269 @@ h3 Header with Horizontal Navbar
-        span.chi-header__title.-d--none.-d-xl--flex Control Center
+        span.chi-header__title.-d--none.-d-xl--flex Application Title
       .chi-header__start
         .-d--flex.-d-lg--none
           button#drawer-trigger-1.chi-button.-icon.-flat.-light.chi-drawer__trigger(data-target='#drawer-1', aria-label='Toggle navigation')
             .chi-button__content
               i.chi-icon.-sm--2.icon-menu
       .chi-header__end
-        .toolbar__info
-          .-d--none.-d-md--flex.-align-items-md--center MyLevel3
-          a.chi-button.-icon.-flat.-light.-ml--1(href="#", aria-label="Support")
-            .chi-button__content
-              i.chi-icon.icon-circle-question-outline
-        a.chi-button.-icon.-flat.-light.-ml--1.-d-md--none(href="#", aria-label="User info")
+        a.chi-button.-icon.-flat.-light.-ml--1.-d-md--none(href='#', aria-label='User info')
           .chi-button__content
             i.chi-icon.icon-user
-        a.chi-button.-icon.-flat.-light.-sm.-d--none.-d-md--flex.-ml--1(href="#", aria-label="User")
-          .chi-button__content
-            | Elizabeth
-            i.chi-icon.icon-chevron-down.-ml--1
-        a.chi-button.-icon.-flat.-light.-sm.-d--none.-d-md--flex.-ml--1(href="#", aria-label="Telephone")
+        a.chi-button.-icon.-flat.-light.-d--none.-d-md--flex.-ml--1(href='#', aria-label='User')
           .chi-button__content
-            | 123456789
+            | Item
             i.chi-icon.icon-chevron-down.-ml--1
-    .chi-divider.-colorbar.-d--none.-d-lg--flex
+    .chi-divider.-inverse.-d--none.-d-lg--flex
     nav.chi-header__navbar
       .chi-header__start
         .-d--none.-d-lg--flex
           ul.chi-tabs.-sm
             li.-active
-              a(href='#') Home
+              a(href='#') Item
             li
-              a(href='#') Admin
+              a(href='#') Item
             li
-              a(href='#') Inventory
+              a(href='#') Item
             li
-              a(href='#') Orders
+              a(href='#') Item
             li
-              a(href='#') Repair
+              a(href='#') Item
             li
-              a(href='#') Reports
+              a(href='#') Item
+// Drawer
+.chi-backdrop.-closed
+  .chi-backdrop__wrapper
+    #drawer-1.chi-drawer.-left.-menu.-position--absolute
+      .chi-drawer__header
+        button.chi-button.-icon.-close(aria-label='Close')
+          .chi-button__content
+            i.chi-icon.icon-x
+      .chi-drawer__content
+        .-px--2.-text Drawer content here
+
+h3 Inverse Portal Header
+header.chi-header.-inverse
```

Comment:
@jllr can you help @sabah1910 here. both the portal and impersonation examples must be wrapped in a test div in order for the tests to run properly.

Reply:
@mattnickles Sure